packages/frontend/src/utils/wallet.js

```diff
@@ -959,7 +972,7 @@ export default class Wallet {
             CONFIG.NETWORK_ID
         );
         const blockNumberSignature = Buffer.from(signed.signature).toString('base64');
-        return { blockNumber, blockNumberSignature };
+        return { blockNumber, blockNumberSignature, publicKey: signed.publicKey.toString() };
```

Comment:
Added public key to the original method because it wasn't used anywhere in the code